pkg/artifactory/resource_artifactory_local_generic_repository.go

```diff
@@ -0,0 +1,17 @@
+package artifactory
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceArtifactoryLocalGenericRepository(pkt string) *schema.Resource {
```

Comment:
Can we rename `pkt` to `packageType`? I prefer more verbose arg/variable names.

Reply:
sure. will do.